android/src/main/java/com/tailscale/ipn/ui/view/ExitNodePicker.kt

```diff
@@ -116,36 +78,43 @@ fun ExitNodePicker(
   }
 }
 
-@Composable
-fun ListHeading(label: String, indent: Dp = 0.dp) {
-  ListItem(
-      modifier = Modifier.padding(start = indent),
-      headlineContent = { Text(text = label, style = MaterialTheme.typography.titleMedium) })
-}
-
 @Composable
 fun ExitNodeItem(
     viewModel: ExitNodePickerViewModel,
     node: ExitNodePickerViewModel.ExitNode,
-    indent: Dp = 0.dp
 ) {
   Box {
+    // TODO: add disabled styling
     ListItem(
-        modifier = Modifier.padding(start = indent).clickable { viewModel.setExitNode(node) },
+        modifier =
+            Modifier.clickableOrGrayedOut(enabled = node.online) { viewModel.setExitNode(node) },
         headlineContent = { Text(node.city.ifEmpty { node.label }) },
+        supportingContent = { if (!node.online) Text(stringResource(R.string.offline)) },
         trailingContent = {
           Row {
             if (node.selected) {
-              Icon(Icons.Outlined.Check, contentDescription = stringResource(R.string.more))
-            } else if (!node.online) {
-              Spacer(modifier = Modifier.width(8.dp))
-              Text(stringResource(R.string.offline))
+              Icon(Icons.Outlined.Check, contentDescription = stringResource(R.string.selected))
             }
           }
         })
   }
 }
 
+@Composable
+fun MullvadItem(nav: ExitNodePickerNav, count: Int, selected: Boolean) {
```

Comment:
Nit: We should probably use a more generic term than "Mullvad" - considering that these are just exit nodes and, in theory, could be provided by any number of sources.

Reply:
We say `Location-based exit nodes` on other platforms. Mullvad happens to be one of the options for it.